src/compas_fab/backends/kinematics/utils.py

```diff
@@ -2,16 +2,11 @@
 
 
 def get_smaller_angle(angle):
-    while angle > 2 * math.pi:
-        angle -= 2 * math.pi
-    while angle < -2 * math.pi:
-        angle += 2 * math.pi
-    if math.fabs(angle - 2 * math.pi) < math.fabs(angle):
-        return angle - 2 * math.pi
-    elif math.fabs(angle + 2 * math.pi) < math.fabs(angle):
-        return angle + 2 * math.pi
-    else:
-        return angle
+    angle = angle % (2 * math.pi)
```

Comment:
Isn't the result of mod always nonnegative? do you mean to swap this line with the next?

Reply:
ah! true.. I forgot that mod returns positive.. that makes it even easier.